tests/package-custom-coverage-test.js

```diff
@@ -234,9 +234,8 @@ describeApplication('PackageCustomCoverage', () => {
           });
 
           it('rejects invalid begin date', () => {
-            convergeOn(() => {
+            return new Convergence().once(() => {
```

Comment:
_How did I not notice this before?_

The previous `it` that was here is a false positive. It never returns the promise it creates so it's essentially the same thing as `it('passes without a body', () => {})`.

We should just remove the convergence here because `it` is already convergent.